docs/user-guide/annotations.md

```diff
@@ -1,66 +1,65 @@
-# Adding Annotations 
+# Annotations 
 ********************
 
 [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) lets you enrich the map with special features which expose additional information, mark particular position on the map and so on.
 Those features make up the so called **Annotations** layers.
 
-Starting from a new map or an already existing one:
+Starting from a new map or an already existing one, the editor can access the **Annotations** <img src="../img/button/annotations2.jpg" class="ms-docbutton" style="max-height:30px;"/> button from the [TOC](toc.md) <img src="../img/button/show-layers.jpg" class="ms-docbutton" style="max-height:30px;"/> panel on the top-left corner of the map viewer. The annotation panel will open:
 
-* **Click** on the *Burger menu* button <img src="../img/button/burger.jpg" class="ms-docbutton" /> from the main menu bar.
-* **Click** on the *Annotations* option from the list <img src="../img/button/annotation-option.jpg" class="ms-docbutton" style="max-height:20px;"/>.
+<img src="../img/annotations/annotation_tool.jpg" class="ms-docimage" style="max-width:500px;" />
```

Comment:
no need for this to be so long in height
![image](https://user-images.githubusercontent.com/11991428/191512706-b76544b2-070c-4fd3-a320-f396cfde590c.png)